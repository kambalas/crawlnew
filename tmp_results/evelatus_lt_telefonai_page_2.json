[
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Pro 128GB White Titanium |",
    "price": "1257.45",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple MOBILE PHONE IPHONE 16 PLUS/256GB WHITE MXWV3",
    "price": "1230.50",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple MOBILE PHONE IPHONE 16 PLUS/256GB PINK MXY13",
    "price": "1230.50",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 15 Plus 256GB Green",
    "price": "1219.18",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 15 Pro 256GB Blue Titanium",
    "price": "1199.00",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 15 Pro 256GB Black Titanium",
    "price": "1199.00",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Plus 128GB Pink |",
    "price": "1160.96",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 256GB Black |",
    "price": "1153.75",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 256GB White |",
    "price": "1153.75",
    "error": false
  },
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  }
]