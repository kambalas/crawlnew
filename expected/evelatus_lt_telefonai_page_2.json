[
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Pro 128GB White Titanium",
    "price": "1257.45",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple MOBILE PHONE IPHONE 16 PLUS/256GB WHITE MXWV3",
    "price": "1230.50",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple MOBILE PHONE IPHONE 16 PLUS/256GB PINK MXY13",
    "price": "1230.50",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 15 Plus 256GB Green",
    "price": "1219.18",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 15 Pro 256GB Blue Titanium",
    "price": "1199.00",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 15 Pro 256GB Black Titanium",
    "price": "1199.00",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Plus 128GB Pink",
    "price": "1160.96",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 256GB Black",
    "price": "1153.75",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 256GB White",
    "price": "1153.75",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple MOBILE PHONE IPHONE 16 PRO MAX/256GB NATURAL TIT. MYWY3",
    "price": "1501.10",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Pro Max | White Titanium | 6.9 \" | Super Retina XDR...",
    "price": "1497.22",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Pro Max 256GB Natural Titanium |",
    "price": "1497.22",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Pro | Desert Titanium | 6.3 \" | Super Retina XDR...",
    "price": "1407.33",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Pro | Natural Titanium | 6.3 \" | Super Retina XDR...",
    "price": "1257.45",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Pro | Black Titanium | 6.3 \" | Super Retina XDR...",
    "price": "1257.45",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Pro | 128GB White Titanium |",
    "price": "1257.45",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Plus | Pink | 6.7 \" | Super Retina XDR display...",
    "price": "1290.50",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Plus 256GB Ultramarine |",
    "price": "1294.09",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Plus 256GB Black |",
    "price": "1294.09",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Plus 256GB White |",
    "price": "1230.50",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 16 Plus 256GB Pink |",
    "price": "1230.50",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 15 Pro Max/256GB BLUE MU7A3PX/A",
    "price": "1323.25",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple MOBILE PHONE IPHONE 15 PRO MAX/256GB BLUE MU7A3PX/A",
    "price": "1323.25",
    "error": false
  },
  {
    "name": "Išmanusis telefonas Apple iPhone 15 Pro Max/256GB N.TITANIUM MU793",
    "price": "1323.25",
    "error": false
  }
]